windows/ReactNativeWebView/ReactWebView.cpp

```diff
@@ -132,6 +139,16 @@ namespace winrt::ReactNativeWebView::implementation {
         PostMessage(winrt::hstring(args.Value()));
     }
 
+    void ReactWebView::OnPremissionRequested(winrt::WebView const& webView, winrt::WebViewPermissionRequestedEventArgs const& args)
+    {
+      if (args.PermissionRequest().PermissionType() == WebViewPermissionType::ImmersiveView) {
+        // WebGL permission request
+        // https://docs.microsoft.com/en-us/microsoft-edge/webvr/webvr-in-webview
+        args.PermissionRequest().Allow();
+      }
+      args.PermissionRequest().Deny();
```

Comment:
It's possible there could be other pieces of the webview that want to register for webview permission request callbacks (even though that's not the case today), I wonder if it would be better to not deny these and let other listeners deal with it; this might result in some permission requests being raised more than once but would give the control more flexibility to decentralize access.

Reply:
a5933a87a3d638bca6c4d49321d12b6755fe69ad